haiku-apps/2dphysicsdemo/2dphysicsdemo-22.5.98.recipe

```diff
@@ -0,0 +1,45 @@
+SUMMARY="2D Physics demo"
+DESCRIPTION="2D Physics Test Program is a cheesy test harness for 2D physics."
+HOMEPAGE="https://github.com/HaikuArchives/2dPhysicsDemo"
+COPYRIGHT="Chris Hecker, Andreas Kaenner"
+LICENSE="unknown"
+REVISION="1"
+srcGitRev="ca884c1af8bee25d71389df85001757bef4a3a6a"
+SOURCE_URI="https://github.com/HaikuArchives/2dPhysicsDemo/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="fac4518e4c6f5497cf3ef51a6f3ad2091e39844e75825da379ed16ca7cadb7b7"
+SOURCE_FILENAME="2dPhysicsDemo-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="2dPhysicsDemo-$srcGitRev"
+PATCHES="2dphysicsdemo-$portVersion.patchset"
```

Comment:
@bach5000, it would be great if you could upstream this patch. Would you do that for us? Thanks!